lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -228,6 +235,59 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Split $this->_bindedValues into those values that need to be sent using mysqli::send_long_data()
+     * and those that can be bound the usual way.
+     *
+     * @return array
+     */
+    private function separateLongData()
```

Comment:
Would it make sense to call it `separateBoundValues()` instead? We want to take all bound values and separate them in two groups, one of which is the streams which need to be sent using `mysqli::send_long_data()`.